core/src/test/java/nl/nn/adapterframework/pipes/PipeTestBase.java

```diff
@@ -1,111 +1,57 @@
 package nl.nn.adapterframework.pipes;
 
-import org.apache.logging.log4j.Logger;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.rules.ExpectedException;
-
 import nl.nn.adapterframework.configuration.ConfigurationException;
-import nl.nn.adapterframework.configuration.ConfigurationWarnings;
-import nl.nn.adapterframework.core.Adapter;
+import nl.nn.adapterframework.core.ConfiguredTestBase;
 import nl.nn.adapterframework.core.IPipe;
 import nl.nn.adapterframework.core.PipeForward;
-import nl.nn.adapterframework.core.PipeLine;
-import nl.nn.adapterframework.core.PipeLine.ExitState;
-import nl.nn.adapterframework.core.PipeLineExit;
 import nl.nn.adapterframework.core.PipeLineSession;
 import nl.nn.adapterframework.core.PipeRunException;
 import nl.nn.adapterframework.core.PipeRunResult;
 import nl.nn.adapterframework.core.PipeStartException;
 import nl.nn.adapterframework.stream.Message;
-import nl.nn.adapterframework.testutil.TestConfiguration;
-import nl.nn.adapterframework.util.LogUtil;
-
-public abstract class PipeTestBase<P extends IPipe> {
-	protected Logger log = LogUtil.getLogger(this);
 
-	protected PipeLineSession session = new PipeLineSession();
+public abstract class PipeTestBase<P extends IPipe> extends ConfiguredTestBase {
```

Comment:
Kunnen we niet beter FixedForwardPipeTestBase en AbstractPipeTestBase gebruiken?